environment

```diff
@@ -30,6 +30,13 @@ _select () {
 # https://github.com/boto/boto3/issues/1179#issuecomment-569940530
 #
 _preauth () {
+    if ! gcloud auth application-default print-access-token >/dev/null 2>&1; then
+        gcloud auth application-default login --project="$GOOGLE_PROJECT"
+    else
+        gcloud auth application-default set-quota-project "$GOOGLE_PROJECT"
```

Comment:
Please add a comment above, briefly explaining what this does.

Reply:
I did some more reading and I think it's more correct (and clearer) to set the `core/project` instead of just the quota project. I think this absolves the need for a comment (?)